tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -254,18 +275,21 @@ def generate_files(list, args):
 
     # Process props and targets files
     if is_windowsai_package:
-        # Process props file
-        windowsai_props = os.path.join(args.sources_path, 'csharp', 'src', 'Microsoft.ML.OnnxRuntime',
```

Comment:
>OnnxRuntime [](start = 89, length = 11)

Why was it Microsoft.ML.Onnxruntime before?

Reply:
There was only 1 folder before, so all of the packaging props/targets ended up there. Now i created separate folders for WindowsAI/WindowsAI Interop